packages/lab/src/components/Flow/Node/Node.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  ExtractNames,
+  HvBaseProps,
+  HvButton,
+  HvTooltip,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+import { Down, Info, Up } from "@hitachivantara/uikit-react-icons";
+import { getColor } from "@hitachivantara/uikit-styles";
+import { useEffect, useState } from "react";
+import { Handle, NodeProps, Position, useReactFlow, useStore } from "reactflow";
+import { useFlowContext } from "../FlowContext/FlowContext";
+
+import { HvFlowNodeInput, HvFlowNodeOutput, HvFlowNodeParam } from "../types";
+import { staticClasses, useClasses } from "./Node.styles";
+import ParamRenderer from "./Parameters/ParamRenderer";
+
+export { staticClasses as flowNodeClasses };
+
+export type HvFlowNodeClasses = ExtractNames<typeof useClasses>;
+
+export interface HvFlowNodeProps extends Omit<HvBaseProps, "id">, NodeProps {
+  /** Node title */
+  title: string;
+  /** Node description */
+  description: string;
+  /** Node expanded */
+  expanded?: boolean;
+  /** Node inputs */
+  inputs?: HvFlowNodeInput[];
+  /** Node outputs */
+  outputs?: HvFlowNodeOutput[];
+  /** Node parameters */
+  params?: HvFlowNodeParam[];
+  /** A Jss Object used to override or extend the styles applied to the component. */
+  classes?: HvFlowNodeClasses;
+}
+
+const isInputConnected = (id, type, idx, edges) => {
+  if (type === "target") {
+    return edges.some(
+      (e) => e.target === id && e.targetHandle === idx.toString()
+    );
+  }
+  if (type === "source") {
+    return edges.some(
+      (e) => e.source === id && e.sourceHandle === idx.toString()
+    );
+  }
+};
+
+export const HvFlowNode = ({
+  id,
+  type,
+  title,
+  description,
+  expanded = false,
+  params,
+  classes: classesProp,
+  className,
+}: HvFlowNodeProps) => {
+  const [showParams, setShowParams] = useState(expanded);
+  const reactFlowInstance = useReactFlow();
+
+  const { classes, cx, css } = useClasses(classesProp);
+
+  const { nodeGroups, nodeTypes } = useFlowContext();
+  const edges = useStore((s) => s.edges);
+  const nodes = useStore((s) => s.getNodes());
+
+  const node = nodes.find((n) => n.id === id);
+
+  const groupId = nodeTypes?.[type].meta?.groupId;
+  const groupLabel = nodeTypes?.[type].meta?.label;
```

Comment:
Sorry, in my previous review I induced you in error 😅 We should get the group label by doing:

```
const groupLabel = groupId && nodeGroups && nodeGroups[groupId].label)
```

Otherwise, we'll get the node type label and that's not what we want.